plugins/inventory/scaleway.py

```diff
@@ -1,36 +1,38 @@
-# Copyright (c) 2017 Ansible Project
+# Copyright: (c) 2017 Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
 
 __metaclass__ = type
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
     name: scaleway
     author:
       - Remy Leone (@sieben)
     short_description: Scaleway inventory source
     description:
-        - Get inventory hosts from Scaleway
+        - Get inventory hosts from Scaleway.
     options:
         plugin:
             description: token that ensures this is a source file for the 'scaleway' plugin.
             required: True
             choices: ['scaleway', 'community.general.scaleway']
         regions:
-            description: Filter results on a specific Scaleway region
+            description: Filter results on a specific Scaleway region.
             type: list
             default:
                 - ams1
                 - par1
                 - par2
                 - waw1
         tags:
-            description: Filter results on a specific tag
+            description: Filter results on a specific tag.
             type: list
         oauth_token:
             required: True
-            description: Scaleway OAuth token.
+            description:
+            - Scaleway OAuth token.
+            - More details on how to generate L(token, https://www.scaleway.com/en/docs/generate-api-keys/).
```

Comment:
That way it's easier to use with a screen reader.
```suggestion
            - More details on L(how to generate token, https://www.scaleway.com/en/docs/generate-api-keys/).
```